Rewrite the 'PivotGrid - Running Totals' demo description (#1108)
DevExpress/devextreme-demos: JSDemos/Demos/PivotGrid/RunningTotals/description.md

> The PivotGrid can calculate running totals by rows or columns. To specify the direction, set a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to *"row"* or *"column"*. This property is available only for data fields.

Reviewer: > To specify the direction

This phrase indirectly implies a choice between rows and columns as possible sources for running total calculations. I would have rephrased this in a more direct manner.

> The PivotGrid can calculate running totals by rows or columns. Assign *"row"* or *"column"* to a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to switch between directions available for running totals calculation. This property is available only for data fields.

Reviewer 2: `Assign *"row"* or *"column"* to a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to switch between directions available for running totals calculation.`

I'm not sure if "switch between directions available" is the best way to phrase this. What do you think about the following?

`Assign *"row"* or *"column"* to a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to specify how running totals are calculated.`

> You can use the [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property of the to enable calculation of running totals in the PivotGrid. Specify this property in a field object in the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array (see the `Running Total` field in this demo). Assign *"column"* or *"row"* to this property to calculate running totals for columns or rows respectively.

Reviewer 3: ```suggestion
You can use the fields[].[runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to enable calculation of running totals for a specific PivotGrid field. Set this property to *"column"* or *"row"* to calculate totals by columns or rows respectively.
```

Reviewer 4: Before you get to the properties, you should begin an introductory sentence to create more natural flow in the text.

```suggestion
The PivotGrid can calculate running totals by rows or columns. To specify the direction, set a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to *"row"* or *"column"*. This property is available only for data fields.
```

> Running totals are calculated from scratch for each [group](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within a row or column. If you want to disable this behavior, assign **true** to the [allowCrossGroupCalculation](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#allowCrossGroupCalculation) property. In this case, the calculation continues across all groups.

Reviewer: > If you want to disable this behavior

This phrase tells us no useful information by itself. Moreover, you actually convey useful information in the last sentence:

> In this case, the calculation continues across all groups.

```suggestion
Running totals are calculated from scratch for each [group](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within a row or column. You can assign **true** to the [allowCrossGroupCalculation](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#allowCrossGroupCalculation) property to calculate running totals across all groups.
```

> You can calculate running totals for adjacent [groups](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within one row or column. Assign **true** to the [allowCrossGroupCalculation](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#allowCrossGroupCalculation) property to enable this behavior as shown in this demo.

Reviewer 4: > You can calculate running totals for adjacent [groups](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within one row or column.

You need to first describe how is this a change to the default behavior.

```suggestion
Running totals are calculated from scratch for each [group](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#groupName) within a row or column. If you want to disable this behavior, assign **true** to the [allowCrossGroupCalculation](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#allowCrossGroupCalculation) property. In this case, the calculation continues across all groups.
```

The PivotGrid can calculate running totals by rows or columns. 

> Assign *"row"* or *"column"* to a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to define in which direction to calculate running totals. This property is available only for data fields.

Reviewer: ```suggestion
Assign *"row"* or *"column"* to a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to switch between directions available for running totals calculation. This property is available only for data fields.
```
